diff --git a/node_modules/@dashersw/node-discover/lib/network.js b/node_modules/@dashersw/node-discover/lib/network.js
index f2ca8e1..134353e 100644
--- a/node_modules/@dashersw/node-discover/lib/network.js
+++ b/node_modules/@dashersw/node-discover/lib/network.js
@@ -4,5 +4,5 @@
     os = require('os'),
     EventEmitter = require('events').EventEmitter,
     util = require('util'),
-    uuid = require('uuid/v4'),
+    uuid = require('crypto').randomUUID,
     nodeVersion = process.version.replace('v','').split(/\./gi).map(function (t) { return parseInt(t, 10) });
